### PR TITLE
(NOBIDS) Improve error handling for rewards

### PR DESCRIPTION
**Pull request by recy21**
### <samp>🤖 Generated by Copilot at b92a1ad</samp>

Improve error logging in `rewards.go`. Add `utils.LogError` to capture database errors when fetching validators data.

### CODE REVIEW

**Review by D13ce — COMMENTED** [review]
Change looks good, I have left a comment though. ptal

File: services/rewards.go [hunk 1]
```diff
@@ -300,7 +300,7 @@ func getValidatorDetails(validators []uint64) [][]string {
 		 where validatorindex=ANY($1)
 		 order by validatorindex asc`, validatorFilter)
 	if err != nil {
-		logger.Errorf("error getting validators Data: %v", err)
+		utils.LogError(err, "error getting validators data", 0, map[string]interface{}{"validators": fmt.Sprintf("%v", validators)})
```
[the remark this answers is not in the document]
Comment (recy21) [reply services/rewards.go:303]: @D13ce as this is an end-less, as long there are non changed errors in anywhere in code base.

[the remark this answers is not in the document]
Comment (D13ce) [reply services/rewards.go:303]: As discussed, I have changed the second error too.

**Review by D13ce — APPROVED** [review]
I have made a commit to change the second error too. ptal
If everything's fine, feel free to merge.

**Pull request merged**